From 1b98a931c3bf8daccc48cd618335ff35e3d382da Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Tue, 12 Oct 2021 23:46:42 +0800
Subject: [PATCH] BiF/CMakeLists.txt: remove opt from DEPENDS

Otherwise it starts failing with:

| ninja: error: 'IGC/VectorCompiler/lib/BiF/opt', needed by 'IGC/VectorCompiler/lib/BiF/VCBiFPrintfOCL32.opt.bc', missing and no known rule to make it

We don't need to explicitly make sure opt is built when
using prebuilt binaries.

Upstream-Status: Inappropriate

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 IGC/VectorCompiler/lib/BiF/cmake/Functions.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: git/IGC/VectorCompiler/lib/BiF/cmake/Functions.cmake
===================================================================
--- git.orig/IGC/VectorCompiler/lib/BiF/cmake/Functions.cmake
+++ git/IGC/VectorCompiler/lib/BiF/cmake/Functions.cmake
@@ -121,7 +121,7 @@ function(vc_build_bif RES_FILE CMCL_SRC_
     COMMENT "vc_build_bif: Translating CMCL builtins:  ${BIF_CLANG_BC_NAME_FINAL} -> ${BIF_OPT_BC_NAME}"
     COMMAND CMCLTranslatorTool ${OPT_OPAQUE_ARG} -o ${BIF_CMCL_BC_PATH} ${BIF_CLANG_BC_PATH_FINAL}
     COMMAND ${LLVM_OPT_EXE} ${OPT_OPAQUE_ARG} --O2 -o ${BIF_OPT_BC_PATH} ${BIF_CMCL_BC_PATH}
-    DEPENDS CMCLTranslatorTool ${LLVM_OPT_EXE} ${OPT_BC_DEPENDS})
+    DEPENDS CMCLTranslatorTool ${BIF_CLANG_BC_PATH_FINAL})
 
   add_custom_target(${TARGET_NAME}
     DEPENDS ${BIF_OPT_BC_PATH}
